### PR TITLE
NAPSSPO-597 - add sops to tssc-base image

### PR DESCRIPTION
**Pull request by itewk**
#### What is this PR About?
Describe the contents of the PR

### CODE REVIEW

File: tssc-base/Dockerfile.ubi8 [hunk 1]
```diff
@@ -12,6 +13,10 @@ MAINTAINER Red Hat TSSC <napsspo+tssc@redhat.com>
 COPY --from=origin-cli /usr/bin/oc /usr/bin/oc
 COPY --from=origin-cli /usr/bin/kubectl /usr/bin/kubectl
 
+# arguments
+# NOTE: arguments used in sub stage have to be in that stage (at least when building with docker, buildah doesn't seem to have a problem either way)
+ARG SOPS_RPM="https://github.com/mozilla/sops/releases/download/v3.6.1/sops-3.6.1-1.x86_64.rpm"
```
Comment (HunterGerlach) [inline tssc-base/Dockerfile.ubi8:18]: Do we want to do a specific version here or get the latest tag to make sure we have the most recent patches?

Comment (itewk) [reply tssc-base/Dockerfile.ubi8:18]: @HunterGerlach do you know of anyway to build a url to a latest release on github?

Comment (HunterGerlach) [reply tssc-base/Dockerfile.ubi8:18]: Not off-hand

**Review by HunterGerlach — APPROVED** [review]

File: tssc-base/Dockerfile.ubi8 (hunk 1 above)
Comment (HunterGerlach) [reply tssc-base/Dockerfile.ubi8:18]: Something like the following _could_ work: `wget "https://github.com/mozilla/sops/releases/latest/download/sops-$(curl --silent "https://api.github.com/repos/mozilla/sops/releases/latest" | grep '"tag_name":' | sed -E 's/.*"v([^"]+)".*/\1/')-1.x86_64.rpm"`

Comment (itewk) [reply tssc-base/Dockerfile.ubi8:18]: .... if its cool with you i am going to stick with the hard coded...

**Review by kharyam — DISMISSED** [review]
Looks good!

**Comment by itewk** [comment]
replacing with https://github.com/rhtconsulting/tssc-containers/pull/24

**Pull request closed without merging**